beep/features/featurizer_helpers.py

```diff
@@ -886,3 +887,57 @@ def get_fractional_quantity_remaining(
     )
     summary_diag_cycle_type.columns = ["cycle_index", "fractional_metric"]
     return summary_diag_cycle_type
+
+
+def get_fractional_quantity_remaining_nx(
+    processed_cycler_run, metric="discharge_energy", diagnostic_cycle_type="rpt_0.2C"
+):
+    """
+    Similar to get_fractional_quantity_remaining()
+    Determine relative loss of <metric> in diagnostic_cycles of type <diagnostic_cycle_type> after 100 regular cycles
```

Comment:
Can you also update the docstring here: need to remove "after 100 regular cycles"